src/Windowing/Silk.NET.Windowing.Common/Internals/WindowImplementationBase.cs

```diff
@@ -177,7 +177,14 @@ public WindowBorder WindowBorder
         }
 
         // Other property implementations
-        public bool TransparentFramebuffer => ExtendedOptionsCache.TransparentFramebuffer;
-        public Rectangle<int> BorderSize => IsInitialized ? CoreBorderSize : default;
+        public bool           TransparentFramebuffer => ExtendedOptionsCache.TransparentFramebuffer;
+
+        public bool TopMost
+        {
+            get => ExtendedOptionsCache.TopMost;
+            set => ExtendedOptionsCache.TopMost = value;
+        }
```

Comment:
Can you look at how we've done it for WindowBorder and friends (i.e. we have WindowBorder and a CoreWindowBorder, where the latter is only called if the view is initialized and goes straight to GLFW)

Reply:
Ok, fixed it on the latest commit